### PR TITLE
Update shadow doc to include a blub about selections

### PR DESCRIPTION
**Pull request by Geethree**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
I didn't stumble across the EA doc with the selection process. @guineveresaenger kindly pointed me in the right direction! (Thanks!).

This adds a new Q/A about the process and links to the relevant docs.

### CODE REVIEW

File: release-team/shadows.md [hunk 1]
```diff
@@ -107,3 +107,7 @@ A: This happens frequently; usually the Shadow is simply dropped from the roster
 *Q: What if the Role Lead can't continue, and one of the Shadows needs to take over?*
 
 A: The Release Lead, the EA, and the outgoing Role Lead (if possible) will discuss the matter and select one of the Shadows to become the new Role Lead.  This is one of the reasons we have Shadows.
+
+*Q: How/When are Shadows selected?*
+
+A: The Release Leads work with the EA and are responsible for collecting shadow applications and selecting the the Shadows. Generally this is one week into the release. [Relevant EA docs](./role-handbooks/emeritus-adviser#shadow-selection).
```
Comment (hasheddan) [inline release-team/shadows.md:113]:
```suggestion
A: The Release Leads work with the EA and are responsible for collecting shadow applications and selecting the Shadows. Generally this is one week into the release. [Relevant EA docs](./role-handbooks/emeritus-adviser#shadow-selection).
```

**Review by hasheddan — COMMENTED** [review]
Thanks for the update @Geethree! Could you fix the minor typo below and also sign CLA? :)

File: release-team/shadows.md (hunk 1 above)
Comment (Geethree) [reply release-team/shadows.md]: Done. And Done. @hasheddan !

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by guineveresaenger** [comment]
ha! came here too late! 🎉 yay and welcome to Kubernetes!

**Pull request merged**